templates/config/credentials.template.json

```diff
@@ -3,10 +3,10 @@
   "stores": [
     {
       "name": "${AM_STORE_NAME}",
-      "endpoint": "http://${S3_SERVER}:${S3_PORT}/",
+      "endpoint": "${S3_SERVER}:${S3_PORT}",
```

Comment:
Can we also have `"proxyUrl": "${S3_EXT_IP}:${S3_EXT_PORT}",` or perhaps `"proxyHost": "${S3_EXT_IP}:${S3_EXT_PORT}",` as well, to avoid any confusions?

Reply:
I think this is wrong because the protocol is important for a proxy url. I wasn't aware that the minio sdk doesn't not accept full urls. To be honest, this is not an install script bug anymore since the configuration is done automatically anyway. We can move this discussion on the AM project if needed.